db/schema.rb

```diff
@@ -24,20 +24,9 @@
     t.string   "category"
   end
 
-  create_table "assets", force: :cascade do |t|
-    t.integer  "employees_id"
-    t.integer  "laptops_id"
-    t.datetime "created_at",       null: false
-    t.datetime "updated_at",       null: false
-    t.string   "item_model"
-    t.string   "item_description"
-    t.integer  "item_price"
-    t.string   "item_attributes"
-  end
-
   create_table "employees", force: :cascade do |t|
-    t.string   "first_name"
-    t.string   "last_name"
+    t.string   "fname"
+    t.string   "lname"
```

Comment:
Why did you change this?

fname and lname are not good. Should be first_name and last_name